ansible/roles/cartodb.app/templates/cartodb_server.conf.j2

```diff
@@ -0,0 +1,17 @@
+# WARNING: This file is auto-generated by Ansible. Any changes will
+# be overwritten the next time ansible is run.
+
+description "Start the CartoDB rails development server"
+
+stop on runlevel [!2345]
+
+respawn
+# Kill respawn if more than 5 restarts in 20 mins
+respawn limit 5 20
+
+script
+    APP_ROOT={{ cartodb_dir }}
+
+    cd $APP_ROOT
+    sudo -u {{ ansible_ssh_user }} RAILS_ENV={{ cartodb_environment }} bundle exec rails server
```

Comment:
Just tried this, for some reason it doesn't pick up the locally installed bundle. Made the chdir change specified above, can investigate this later